Library/ViewModels/ProjectPamphletViewModel.swift

```diff
@@ -70,8 +70,14 @@ ProjectPamphletViewModelOutputs {
     let refTag = self.refTagProperty.signal
       .map { $0.map(cleanUp(refTag:)) }
 
-    let liveStreamEventsFetch = project
-      .take(first: 1)
+    let projectWhenViewAppears = project.takeWhen(
+      self.viewWillAppearAnimated.signal
+    )
+
+    let liveStreamEventsFetch = Signal.merge(
+      project.take(first: 1),
+      projectWhenViewAppears
+      )
```

Comment:
nice nice! i think you could even just inline this without the intermediate signal:

```swift
let liveStreamEventsFetch = project
  .take(first: 1)
  .takeWhen(self.viewWillAppearAnimated.signal)
  .switchMap { ...
```